packages/components/src/Autocomplete/Autocomplete.mdx

```diff
@@ -154,3 +154,58 @@ using either promises or `await` in `getOptions`.
     }
   }}
 </Playground>
+
+## Autocomplete with section headings, description and extra details
+
+You can specify an array of sections which contains options objects.
+
+<Playground>
+  {() => {
+    const options = [
+      { label: "Ships", heading: true },
+      {
+        value: 1,
+        label: "Sulaco",
+        description: "They mostly come at night, mostly.",
+        details: "LV-426"
+      },
+      { value: 2, label: "Nostromo", details: "LV-426" },
+      { value: 3, label: "Serenity", description: "I aim to misbehave." },
+      { value: 4, label: "Sleeper Service" },
+      { value: 5, label: "Enterprise" },
+      {
+        value: 6,
+        label: "Enterprise-D",
+        description: "Tea, earl grey, hot.",
+        details: "NCC-1701D"
+      },
+      { label: "Planets", heading: true },
+      { value: 7, label: "Endor" },
+      { value: 8, label: "Vulcan" },
+      { value: 9, label: "Bespin" },
+      { value: 10, label: "Tatooine" },
+      { value: 11, label: "Enterprise" },
+      { value: 12, label: "Ariel" }
+    ];
+    const [value, setValue] = useState();
+    return (
+      <Autocomplete
+        value={value}
+        sections={true}
```

Comment:
Hopefully we can find a structure for the data passed in (I guess through `getOptions`) - that means the component can automatically detect the difference. Then we can avoid having to specify it through a prop, and just have it be based on the data structure. Thoughts?

Reply:
Oh... does sections actually get used? I think you might not even be using it...

Reply:
Nope this is cruft! Sorry, gone now.